.env.dist

```diff
@@ -0,0 +1,22 @@
+# This file is a "template" of which env vars needs to be defined in your configuration or in an .env file
+# Set variables here that may be different on each deployment target of the app, e.g. development, staging, production.
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_DEBUG=1
+APP_SECRET=96054558daeafea809d12eb054a9298e
+###< symfony/framework-bundle ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# For a MySQL database, use: "mysql://root@127.0.0.1:3306/symfony?charset=utf8mb4&serverVersion=5.7"
+# Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
+DATABASE_URL="sqlite:///%kernel.project_dir%/var/data/blog.sqlite"
```

Comment:
I agree with @yceruto, it's better to use relative path here. We should avoid to use container parameters in the environment files, because it makes them unshareable with others applications.
For example, I want to create bash script to back up the database:
```sh
#!/usr/bin/env bash

source .env;
mysqldump "$DATABASE_URL"; # It won't work because variable contains the %kernel.project_dir% parameter
```